src/test/e2e/util/ImageUtils.java

```diff
@@ -29,12 +29,12 @@ private static float imgDifference(PImage i0, PImage i1) {
     i1.loadPixels();
     int[] ip1 = i1.pixels;
     for (int n = 0; n < ip0.length; n++) {
-      int pxl0 = ip0[n]
```

Comment:
That's weird. Blame says that I introduced this breaking typo when I merged a large set of codacy style changes -- 

https://github.com/processing-r/Processing.R/blame/733fb86e5a695eb78d26082cb189098f3b901b21/src/test/e2e/util/ImageUtils.java#L32

but then that build passed on the CI server!

https://travis-ci.org/github/processing-r/Processing.R/builds/569579118

I'm not sure how that is possible, unless that build didn't perform the e2e ImageUtils phase at all. Something to look at when setting up Actions.

Reply:
Nevermind, that build actually failed in the log, it just displayed as passed for some reason. ugh.